File: AllSpice.client/src/models/Favorite.js
export class Favorite {
  constructor(data) {
    this.id = data.id;
    this.accountId = data.accountId;
    this.recipeId = data.recipeId;
    this.createdAt = data.createdAt;
    this.updatedAt = data.updatedAt;
  }
}

File: AllSpice/Models/Favorite.cs
using AllSpice.Interfaces;

namespace AllSpice.Models;

public class Favorite : IRepoItem<int>
{
  public int favoriteId { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
  public int RecipeId { get; set; }
  public string AccountId { get; set; }
  public int Id { get; set; }
}